.travis.yml

```diff
@@ -164,6 +169,17 @@ jobs:
          - MATRIX_EVAL="CC=clang-8 && CXX=clang++-8"
       script:
         - ./.github/travis/build.sh
+    - stage: Coverity Scan
+      name: "Coverity Scan"
+      env:
+        - _COVERITY_EMAIL="jeanphilippe.legaul@gmail.com"
+        - _COVERITY_URL="https://scan.coverity.com/download/linux64"
+        - _COVERITY_MD5="d0d7d7df9d6609e578f85096a755fb8f"
```

Comment:
These should probably be changed to those of @vaughnbetz or @KennethKent in the future.

Reply:
email is located in the travis UI when the update is needed